### DISCUSSION TITLE
TLS it not available or unauthenticated but required

### DISCUSSION THREAD

**Discussion by cbrake** [emersion/maddy#479] [Q&A] [answered]
I've observed problems delivering email to several different people I communication with:

```
queue: delivery attempt failed  {"domain":"sbcglobal.net","msg_id":"45d30471","rcpt":"someuser@sbcglobal.net","reason":"TLS it not available or unauthenticated but required","smtp_code":451,"smtp_enchcode":"5.4.0","smtp_msg":"No usable MXs, last err: TLS it not available or unauthenticated but required","target":"remote","tls_err":null,"tls_level":"none"}
```
Changing the following gets email delivered:

`target.remote outbound_delivery.mx_auth.local_policy.min_tls_level = none`

I don't like this, but the chances of getting everyone to fix their email servers is next to zero.

Can you confirm that Maddy does use TLS if it is available on the target server, even with this `none` setting?

**Comment by foxcpp** [comment, marked as answer]
Yes, TLS is always tried and DANE and MTA-STS are still used
